german/Hilfeschrei/Vertrauen/insMaul/reinhauen/reinhauen.md

```diff
@@ -1,3 +1,3 @@
 Der Kobold ist für einen Moment etwas irritiert, tut aber keinen Wank und dann...
 
-und verwandelt sich in einen Frosch.
\ No newline at end of file
+[verwandelt er sich in einen Frosch](../../Frosch/Frosch.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.